CMakeLists.txt

```diff
@@ -125,6 +125,7 @@ option(FLB_IN_DISK            "Enable Disk input plugin"            Yes)
 option(FLB_IN_DOCKER          "Enable Docker input plugin"          Yes)
 option(FLB_IN_DOCKER_EVENTS   "Enable Docker events input plugin"   Yes)
 option(FLB_IN_EXEC            "Enable Exec input plugin"            Yes)
+option(FLB_IN_EVENT_TEST      "Enable Events test plugin"           Yes)
```

Comment:
I think for this release (2.0) we need it as `on` because we need help to troubleshoot event issues if the user enable this feature (most of the users are not able to compile the source code)

Reply:
How about using debug container image to troubleshoot that enabled this feature ?

When we request to use this plugin, we will need to fix concurrency issue.
I think it is hard to debug using release image.
We may need to use another debug image that is enabled `-DSANITIZER_THREAD=on` and some tools.